mysql-test/suite/rpl/t/rpl_blob_key.test

```diff
@@ -0,0 +1,14 @@
+--source include/have_binlog_format_row.inc
+--source include/master-slave.inc
+
+CREATE TABLE t1 (b BLOB, i INT, KEY(b(8))) ENGINE=RocksDB;
```

Comment:
One cannot just use `CREATE TABLE ... ENGINE=ROCKSDB` in an arbitrary testcase. MariaDB is built on many platforms, and on many of these MyRocks storage engine is not supported (or sometimes intentionally not built).  

The testcase needs to go into `rocksdb_rpl` testsuite.